src/assets/lang/en.json

```diff
@@ -1,7 +1,11 @@
 {
-  "header": "OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies.",
-  "stats": "{variablesCount, number} variables and {parametersCount, number} parameters referenced to modelize the french tax-benefit system.",
-  "entityParagraph" : "This {variableLink} applies to the {entityLink} ",
+  "header":"OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies. ",
+  "learnMore":"Learn more.",
```

Comment:
No :wink:

Trailing & leading space in message strings is not a good practice. You should leave it to the message composer (the part that assembles the different strings in the interface) to decide where to put spaces. For example, if you were to enclose the message in some HTML tags, leading or trailing spaces would not be collapsed together.

Once again: 

> My previous comment on a missing space was after the JSON colon

The expectation is:

```diff
-  "learnMore":" Learn more.",
+  "learnMore": "Learn more.",
               ^^
``` 

I'll handle these changes.